gen3/bin/kube-setup-pidgin.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#
+# Deploy pidgin into existing commons
+# This fragment is pasted into kube-services.sh by kube.tf.
+#
+
+_KUBE_SETUP_PIDGIN=$(dirname "${BASH_SOURCE:-$0}")  # $0 supports zsh
+source "${_KUBE_SETUP_PIDGIN}/../lib/kube-setup-init.sh"
```

Comment:
Please use the same sourcing mechanism we're using in the other scripts - it avoids double-sourcing files:
```

source "${GEN3_HOME}/gen3/lib/utils.sh"
gen3_load "gen3/lib/kube-setup-init"
```